cmake/CPack.STGZ_Header.sh.in

```diff
@@ -245,37 +184,66 @@ if [ -z "$SHELL" ];then
       user_sh=$(basename $user_sh)
    fi
 else
-        user_sh=$(basename $SHELL)
+  user_sh=$(basename $SHELL)
 fi
 
 bash_exist=1
-if [ $user_sh = "bash" ];then
-	sh_profile=$HOME/.bash_profile
-	if [ ! -f $sh_profile ];then
-	  bash_exist=0
-	  touch $sh_profile
-	fi
-
-	append_profile=$(grep "${PRODUCT_NAME} environment" $sh_profile)
-
-	if [ -z "$append_profile" ];then
-	  echo '#-------------------------------------------------------------------------------' >> $sh_profile
-	  if [ $? -ne 0 ];then
-	    CUBRID_SH_INSTALLED=0
-	    echo "Please check your permission for file $sh_profile"
-	  else
-	    echo '# set '${PRODUCT_NAME}' environment variables'                                    >> $sh_profile
-	    echo '#-------------------------------------------------------------------------------' >> $sh_profile
-	    echo 'if [ -f $HOME/.cubrid.sh ];then'                                                  >> $sh_profile
-	    echo '. $HOME/.cubrid.sh'                                                               >> $sh_profile
-	    echo 'fi'                                                                               >> $sh_profile
-	    if [ $bash_exist -eq 0 ];then
-	      echo "Notification: $sh_profile is created"
-	    fi
-	  fi
-	fi
-else
-  CUBRID_SH_INSTALLED=0
+case $user_sh in
+	zsh)
+		sh_profile=$HOME/.zshrc
+		;;
+	bash)
+		sh_profile=$HOME/.bash_profile
+		;;
+	sh)
+		sh_profile=$HOME/.profile
+		;;
+	csh | tcsh)
+		sh_profile=$HOME/.cshrc
+		;;
```

Comment:
I think that an error handling routine is required for other shell cases.

Reply:
Thank you for your comment.
Fixed it.